elements/lisk-cryptography/test/bls_specs/verify_bls/different_tag.yml

```diff
@@ -0,0 +1,7 @@
+input:
+  pk: '0xa491d1b0ecd9bb917989f0e74f0dea0422eac4a873e5e2644f368dffb9a6e20fd6e10c1b77654d067c0618f6e5a7f79a'
+  tag: LSK_TX:v1_
+  netId: '0x9ee11e9df416b18bf69dbd1a920442e08c6ca319e69926bc843a561782ca17ee'
+  message: '0xbeaf'
+  signature: '0xa6f889695b4ee393c6ede6af2215019cf9d7e004781b98ea12d6b227212126687ecde1c2a08e38e2d5c18eab2881879102e91ac5f0e9813126d6d68262af149ba6c25ffb88e6688fec49b5199cec863c0eb54110fdb6d92c6570f3ca9c1910b9'
+output: true
```

Comment:
this should not return true

Reply:
The implementation of `verifyBLS` is in next PR and I fixed it there already.